Adyen/Assets/zh-CN.lproj/Localizable.strings

```diff
@@ -73,8 +72,6 @@
 "adyen.button.copy" = "复制";
 "adyen.voucher.introduction" = "感谢您的购买，请使用以下信息完成支付。";
 "adyen.voucher.readInstructions" = "阅读说明";
-"adyen.voucher.saveImage" = "";
-"adyen.voucher.finish" = "";
```

Comment:
This seems odd.. Maybe revert those lines for now - we can figure out what is wrong with them in next PR